src/cluster/autoscale.go

```diff
@@ -391,3 +403,30 @@ func deployAutoscalerChart(cluster CommonCluster, nodeGroups []nodeGroup, kubeCo
 	log.Infof("'%s' %sed", chartName, action)
 	return nil
 }
+
+func getImageVersion(cluster CommonCluster) map[string]string {
+	k8sVersion := "1.12"
+	if c, ok := cluster.(interface{ GetKubernetesVersion() (string, error) }); ok {
+		k8sVersion, _ = c.GetKubernetesVersion()
```

Comment:
This will overwrite the default value of `k8sVersion` (most probably with `""`) when an error happens.

Reply:
Agree that in case of error `k8sVersion` may be reset.

Reply:
fixed